src/Core.UnitTests/CSharpVB/RuleSetGeneratorTests.cs

```diff
@@ -234,6 +234,34 @@ public void RoslynRuleSet_Sonar_Rules_Added()
                 "Warning", "Warning", "None", "None");
         }
 
+        [TestMethod]
+        [DataRow("cs")]
+        [DataRow("vbnet")]
```

Comment:
There currently aren't any taint analysis rules for VB, but they would be excluded too (assuming a VB security analyzer followed the same naming pattern).